apps/web/src/app/(dashboard)/documents/data-table-action-button.tsx

```diff
@@ -72,6 +73,8 @@ export const DataTableActionButton = ({ row }: DataTableActionButtonProps) => {
     window.URL.revokeObjectURL(link.href);
   };
 
+  if (recipient?.role === RecipientRole.CC && isComplete === false) return <div></div>;
```

Comment:
This one sticks out to me as a question but until I run it locally I'm unsure what the question is. Just seems odd to return nothing here.

Reply:
This is just assuming that the CC'er shouldn't be able to view the document until it has been completed.

Reply:
You are absolutely right, I think the thing tripping me up here is that it appears as though a CC'er can see the document in their account. 

We may want this but only once the document is completed, otherwise perhaps they only ever receive the email.

This may balloon the query needed for `findDocuments` which is a concern.